packages/@divvi/mobile/src/keylessBackup/WalletSecurityPrimer.tsx

```diff
@@ -13,16 +13,22 @@ import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
+import { getAppConfig } from 'src/appConfig'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.WalletSecurityPrimer>
 
 function WalletSecurityPrimer({ route }: Props) {
   const { t } = useTranslation()
+  const cloudBackupConfig = getAppConfig().themes?.default?.assets?.cloudBackup
+  const educationImage = cloudBackupConfig?.educationImage
+    ? cloudBackupConfig.educationImage
+    : walletSafe
```

Comment:
```suggestion
  const educationImage = cloudBackupConfig?.educationImage ?? walletSafe
```
would this work here?